kolibri/auth/test/test_users.py

```diff
@@ -60,12 +59,18 @@ class UserSanityTestCase(TestCase):
     Sanity checks basic functionality of user models.
     """
     def setUp(self):
-        self.user = FacilityUser.objects.create(username="mike", first_name="Mike", last_name="Gallaspy")
-        self.do = DeviceOwner.objects.create(username="bar")
-
-    def test_base_user(self):
-        with self.assertRaises(NotImplementedError):
-            BaseUser().is_device_owner()
+        self.dataset = FacilityDataset.objects.create()
+        self.user = FacilityUser.objects.create(
+            username="mike",
+            first_name="Mike",
+            last_name="Gallaspy",
+            password="###",
```

Comment:
Yep -- the tests here don't do anything with authentication, so it shouldn't be necessary; the only reason I needed to set something for the password field was that these tests call `full_clean`, which throws a validation error if the field is empty.